secretmanager/src/test/java/secretmanager/SnippetsIT.java

```diff
@@ -62,12 +62,17 @@ public class SnippetsIT {
   private static final String IAM_USER =
       "serviceAccount:iam-samples@java-docs-samples-testing.iam.gserviceaccount.com";
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static final String LABEL_KEY = "secretmanager";
+  private static final String LABEL_VALUE = "rocks";
+  private static final String UPDATED_LABEL_KEY = "gcp";
+  private static final String UPDATED_LABEL_VALUE = "rock";
```

Comment:
Though I appreciate the choice of labels here, you might want to consider changing them as some of our customers also use these tests in their environment ;)

Reply:
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/secretmanager/src/main/java/secretmanager/UpdateSecret.java#L49

I added these because such labels are already there in the samples ^

Reply:
Let's maybe ignore the precedence in this case and go for something neutral in the values?

Reply:
Fixed